### DISCUSSION TITLE
Videos have incorrect thumbnail

### DISCUSSION THREAD

**Discussion by dustins5523** [photoprism/photoprism#3502] [Q&A]
#### 1. What is not working as documented?

Build 230615-90a18f6e7
HP Elitedesk 800 i5 6600 3.3 ghz Linux Mint 1TB SSD 16gb RAM
 
Just installed photoprism on a fresh install of Linux Mint. I have experience with home servers, but I am no expert, esp with Linux FWIW

I have a library of around 10K pictures and videos. I noticed some videos have incorrect thumbnails. I googled it and found it can be caused by duplicate file names with different extensions ie: edjucnf4.mp4 and edjucnf4.jpg

I found all duplicate names in my library and renamed one file by adding a 1 ie: edjucnf41.mp4 and edjucnf4.jpg. I shutdown the service, restarted it and did a complete reindex and I still have the same issue. A video will have completely different thumbnail than the content of the video. Is there anyway I can fix this?

**Comment by lastzero** [comment]
As PhotoPrism remembers the file hash (so the content), renaming a sidecar file won't remove it from the video stack. You may be able to do this in the frontend with the Unstack button in the files tab, as detailed in our documentation. However, since these are the primary files, the unstack operation may not be available. In that case, you could try to temporarily remove the JPEGs, run index --cleanup or index and then run the cleanup command, and then readd the jpegs to a different directory so you don't even need to rename them.

**Reply by lastzero** [reply]
In a future release, it might be worth improving the frontend, so it's easier to break up unwanted stacks. It's related to this feature request:
- #667
